api/docs/tutorial.dox

```diff
@@ -300,7 +300,7 @@ event_basic_block(void *drcontext, void *tag, instrlist_t *bb,
     dr_mutex_unlock(as_built_lock);
```

Comment:
Thanks for the fix! This looks good to me.

Just a quick note about the PR description and title: please see https://dynamorio.org/page_code_reviews.html#sec_code_review_non_member.

Particularly, it'll be clearer if the PR title is something like: `Fix dr_insert_clean_call callee arg in tutorial`

And the PR description could be:

```
Fixes the callee arg in the dr_insert_clean_call call in the dynamic basic block size tutorial. The callee pointer should be cast to void* for it to compile successfully.
```

Reply:
done